examples/resources/groundcover_logspipeline/import.sh

```
@@ -0,0 +1,2 @@
+# Logs pipeline is a singleton resource. The import ID value is ignored.
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)